drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -174,7 +174,9 @@ def get_form_definition(role:, enrollment_id: nil, project_id: nil)
       project = Hmis::Hud::Project.find_by(id: project_id) if project_id.present?
       project = Hmis::Hud::Enrollment.find_by(id: enrollment_id)&.project if enrollment_id.present?
 
-      Hmis::Form::Definition.find_definition_for_role(role, project: project)
+      record = Hmis::Form::Definition.find_definition_for_role(role, project: project)
+      record.filter_context = { project: project }
```

Comment:
This needs to be applied [here](https://github.com/greenriver/hmis-warehouse/blob/7c19b5f15fcad3adb2302f951fc136c53951399a/drivers/hmis/app/graphql/types/hmis_schema/assessment.rb#L60-L72) too, so it works when opening an existing assessment.